### PR TITLE
Improves docs information structure.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes https://github.com/filecoin-project/filecoin-docs/issues/1692

Complete revamp of the docs organization. This change was specifically for the FVM project.

### CODE REVIEW

File: content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md [hunk 1]
```diff
@@ -13,32 +13,44 @@ weight: 100
...
+## Gas stipend
 
-Iceland next level literally, butcher pok pok gentrify readymade shaman. Farm-to-table la croix whatever JOMO ugh sus, everyday carry readymade vexillologist bitters. +1 blog intelligentsia hashtag umami, celiac vice photo booth. Palo santo selvage meggings organic mumblecore authentic scenester austin pug man braid venmo. Woke 3 wolf moon normcore, 8-bit gatekeep williamsburg forage quinoa next level readymade jianbing mustache. Trust fund swag godard tumblr chicharrones mlkshk vaporware.
+Solidity calls to `address.transfer` and `address.send` grant a fixed gas stipend of 2300 gas to the called contract. Unfortunately, this is EVM gas, not Filecoin gas. To compensate, Filecoin EVM runtime automatically sets the gas limit to 10 million Filecoin gas if it looks like a `transfer` or `send`.
```
Comment (raulk) [inline content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md:33]: Suggest a simplification and a more straightforward conclusion:

> Solidity calls to `address.transfer` and `address.send` grant a fixed gas stipend of 2300 Ethereum gas to the called contract. The Filecoin EVM runtime automatically detects such calls, and sets the gas limit to 10 million Filecoin gas. This is a relatively more generous limit than Ethereum's, but it's future-proof. You should expect the callee to be able to carry out more work than in Ethereum.

File: content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md [hunk 2]
```diff
@@ -13,32 +13,44 @@ weight: 100
...
+## Bare-value sends
 
-Normcore pinterest gluten-free skateboard godard. Cardigan man bun cred locavore etsy ugh vape tousled swag. Sus art party migas kickstarter tattooed activated charcoal pok pok. Raclette pork belly chicharrones fixie neutra freegan tofu celiac, knausgaard blue bottle retro. +1 tattooed pork belly waistcoat.
+In Ethereum, `SELFDESTRUCT` is the only way to send funds to a smart contract without giving the target smart contract a chance to execute code.
 
-Gentrify fixie schlitz +1 90's tousled. Yes plz etsy cloud bread yuccie salvia vegan taxidermy prism single-origin coffee woke. Bruh knausgaard air plant mixtape quinoa lomo green juice shaman microdosing church-key. Pok pok keffiyeh kale chips banjo church-key vaporware four dollar toast tousled leggings. Authentic ramps PBR&B, biodiesel bruh tumblr butcher echo park vice. Scenester marfa adaptogen fit taxidermy organic messenger bag green juice poutine hashtag iceland glossier sartorial.
+In Filecoin, any actor can use `method 0`, also called a bare-value send, to transfer funds to any other actor without invoking the target actor's code.
+
```
Comment (raulk) [inline content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md:53]: Add:

> You can think of this behaviour as having the suggested [`PAY` opcode](https://eips.ethereum.org/EIPS/eip-5920) already implemented in Filecoin.

**Comment by johnnymatthews** [comment]
It looks like Fleek is having some trouble deploying a preview for this PR now. We might have to close this PR and create a new one; seems like the easiest option.

File: content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md (hunk 1 above)
Comment (johnnymatthews) [reply content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md]: Resolved in [a92ed3d](https://github.com/filecoin-project/filecoin-docs/pull/1694/commits/a92ed3d39ff0db9643c4aaf765f47ed7258f6992)

File: content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md (hunk 2 above)
Comment (johnnymatthews) [reply content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md:53]: Resolved in [a92ed3d](https://github.com/filecoin-project/filecoin-docs/pull/1694/commits/a92ed3d39ff0db9643c4aaf765f47ed7258f6992)

**Pull request merged**